nengo/networks/workingmemory.py

```diff
@@ -31,22 +30,20 @@ def InputGatedMemory(n_neurons, dimensions, fdbk_scale=1.0, gate_gain=10,
         nengo.Connection(net.mem.output, net.diff.input, transform=-1)
 
         # feed difference into integrator
-        with mem_config:
-            nengo.Connection(net.diff.output, net.mem.input,
-                             transform=difference_gain)
+        nengo.Connection(net.diff.output, net.mem.input,
+                         transform=difference_gain,
+                         synapse=nengo.Lowpass(difference_synapse))
 
         # gate difference (if gate==0, update stored value,
         # otherwise retain stored value)
         net.gate = nengo.Node(size_in=1)
         nengo.Connection(net.gate, net.diff.neuron_input,
-                         transform=np.ones((n_total_neurons, 1)) * -gate_gain,
-                         synapse=None)
+                         transform=np.ones((n_total_neurons, 1)), synapse=None)
```

Comment:
This should be ```* 10```.  Getting rid of the parameter doesn't mean we should get rid of the effect.